### PR TITLE
Automatically use SSE produces MediaType when @RestStreamElementType is used

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/sse/SseResource.java [hunk 1]
```diff
@@ -93,8 +92,8 @@ public void sseJson2(Sse sse, SseEventSink sink) throws IOException {
 
     @Path("json/multi")
     @GET
-    @Produces(MediaType.SERVER_SENT_EVENTS)
     @RestStreamElementType(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.SERVER_SENT_EVENTS)
```
Comment (cescoffier) [inline extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/sse/SseResource.java:96]: Why is it required here?

**Review by cescoffier — CHANGES_REQUESTED** [review]
just one question.

File: extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/sse/SseResource.java (hunk 1 above)
Comment (geoand) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/sse/SseResource.java:96]: Ah, it got left over. I will fix it later on today

**Comment by geoand** [comment]
PR fixed

**Review by cescoffier — APPROVED** [review]

**Pull request merged**